src/localize.js

```diff
@@ -28,7 +28,7 @@ const DEFAULT_LANG = 'en';
 export default function localize(string, search, replace) {
   const [section, key] = string.split('.');
 
-  const lang = (localStorage.getItem('selectedLanguage') || navigator.language.split('-')[0] || DEFAULT_LANG)
+  const lang = (JSON.parse(localStorage.getItem('selectedLanguage')) || navigator.language.split('-')[0] || DEFAULT_LANG)
```

Comment:
Why do we need this?

Keep in mind, `JSON.parse` can potentially throw an error, so it's better to keep it inside `try..catch`

Reply:
Because HA stores data in JSON format, check please screenshot
![image](https://user-images.githubusercontent.com/43236573/83515422-031abb80-a4de-11ea-8f0b-e0bdd63f032e.png)

Reply:
Got it. Please, wrap `parse` call in try-catch.

Reply:
I think, this is bug fix for #56

Reply:
> Got it. Please, wrap `parse` call in try-catch.

Done

Reply:
I think that why we had the first replace call which was removing single and double quotes. We can probably get rid of that if we are going with the JSON.parse implementation.

Reply:
> We can probably get rid of that if we are going with the JSON.parse implementation.
The "catch" section will not let rid

Reply:
and 
"null" -> "null"
json.parse("null") -> null